README.md

```diff
@@ -19,18 +19,33 @@ Descriptions of de-id tools to go here
 
 Leaderboards for benchmark datasets per modality to go here.
 
-# synthetic_phi
-
-Tool(s) to create DICOM datasets with synthetic PHI from previously de-identified DICOM data. 
+# Installation
 
 To install the requirements
 
 	pip install -r requirements.txt
 
+# ultrasound synthetic_phi
+
+Tool(s) to create DICOM datasets with synthetic PHI from previously de-identified DICOM data. 
+
 To use the tool
 
+    cd synthetic_phi/ultrasound/src
 	python generate_synth_phi.py -i input_dir -o output_dir -m num_synth_dicoms
 
+# whole slide image synthetic_phi
+
+This library allows python applications to deidentify WSI
+
+* iSyntax: deident_isyntax_file(ident_file_path, deident_file_path)
```

Comment:
What is iSyntax and SVS?

Reply:
Those are whole-slide image formats, similar conceptually to DICOM (though significantly different in details).

Reply:
I couldn't find a single good reference for whole slide image formats, so I'm leaving it as-is.